### ISSUE TITLE
Buildings marked as building=yes not being shown

### ISSUE THREAD

**Issue by chebourne** [openmaptiles/openmaptiles#650]
buildings marked with just 'yes' are not coming through to OMT - they _are_ appearing on OSM:

[https://www.maptiler.com/maps/#basic//vector/15.37/-0.849118/52.222926](url)

Notice missing building to NE of Thomas Dascher Way, which is being shown on OSM:

[https://www.openstreetmap.org/way/677231337#map=16/52.2206/-0.8476](url)

Building was added 1st July 2019 - any idea why it's not being shown?

**Comment by chebourne** [comment]
Have posted this query on Maptiler support - appears to be a problem with new buildings added to OSM in the last month. Updating of OMT to reflect this appears to have not happened. Will keep this ticket posted with any feedback from Maptiler/OMT.com

**Comment by chebourne** [comment]
Update from Maptiler:
Hi James,

You are correct, there is a discrepancy between the given date and data, which are there. We had data from the end of July, however, we found an issue on our maps (something changed in OSM, but our software misinterpret it - and this issue affects a major city) and therefore we rolled back. Currently, we have data from 26.6.2019.

The age of data is being always updated when a new planet is rolled up. However, the system doesn't expect downgrades and this is why it shows the date of the latest planet (which was downgraded in the meanwhile).

Our devs are working on the fix, hopefully, it will be fixed with the next data update.
